### PR TITLE
Use full path of python instead of a file name

### PR DESCRIPTION
**Pull request by longyn**
- In case: python not list in the path.

### CODE REVIEW

File: data/CMakeLists.txt [hunk 1]
```diff
@@ -1,6 +1,8 @@
+find_package(PythonInterp REQUIRED)
+
 set(OPENCC_DICT_BIN opencc_dict)
-set(DICT_MERGE_BIN python ${CMAKE_CURRENT_SOURCE_DIR}/scripts/merge.py)
-set(DICT_REVERSE_BIN python ${CMAKE_CURRENT_SOURCE_DIR}/scripts/reverse.py)
+set(DICT_MERGE_BIN ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/scripts/merge.py)
```
Comment (lotem) [inline data/CMakeLists.txt]: Are quotes necessary, in case `PYTHON_EXECUTABLE` is a path containing space character?

Comment (longyn) [reply data/CMakeLists.txt]: Yes, I will make an update.

Comment (lotem) [inline data/CMakeLists.txt]: Refer to usages of the `PYTHON_EXECUTABLE` variable in the CMake module
https://github.com/Kitware/CMake/blob/master/Modules/FindPythonInterp.cmake#L126
Double quotes around the variable were used.

Comment (longyn) [reply data/CMakeLists.txt]: Agree with you. The ```PYTHON_EXECUTABLE``` should be quoted, code updated.

**Comment by lotem** [comment]
Thanks for the pull request.
Changes approved. You may merge it to master any time.

**Pull request merged**